tests/ssl-opt.sh

```diff
@@ -8617,6 +8683,7 @@ requires_config_enabled MBEDTLS_RSA_C
 requires_config_enabled MBEDTLS_ECDSA_C
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_gnutls
+requires_max_content_len 2048
```

Comment:
The DTLS 1.0 test case just below should have the same requirement.

Also goes for the client and openssl test cases below, where similarly the 1.2 version have a requirement but the 1.0 version doesn't.